components/elements/digit-text-area/DigitTextArea.element.jsx

```diff
@@ -21,19 +32,64 @@ const DigitTextArea = ({
   errorMessage,
   disabled,
   rows,
-  rowsMax
+  rowsMax,
+  outline,
+  filled
 }) => (
   <Fill>
-    <FormControl error={error}>
+    <FormControl
+      variant={outline ? "outlined" : filled ? "filled" : "standard"}
+      error={error}
+    >
       <InputLabel>{upperLabel}</InputLabel>
-      <Input
-        name={name}
-        value={value != null ? value : ""}
-        onChange={onChange}
-        onBlur={onBlur}
-        disabled={disabled}
-        rows={rows}
-        rowsMax={rowsMax}
+
+      <DigitIfElseRendering
+        test={outline != null && outline}
+        ifRender={() => (
+          <OutlinedInput
+            name={name}
+            labelWidth={get_tex_size(upperLabel, "1rem Helvetica") * 0.75 + 8}
+            value={value != null ? value : ""}
```

Comment:
`value || ""` is the standard way of writing this. Unless value can be 0 or false, in which case it should be kept as is.